nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2296,7 +2481,11 @@ struct UserPathParam {
     user_name: Name,
 }
 
-/// Fetch a specific built-in system user
+/// Get a user
+///
+/// Returns details of the specified user. If you pass `me` as the `user_name`,
```

Comment:
Is that true?

What if another user's login is `me`?

Reply:
I think we can reserve it, a bunch of apis I've used have this and it's very nice, or at least some way of introspecting yourself

Reply:
The string itself could be like _self _ me whatever

Reply:
I mean im fine with `/session/mew` as long as it has more info about the user, but its nice if its somehow under the same path as other user controls

Reply:
I don't see how we can reserve it because we don't control it -- it comes from the IdP.  If someone signs up with their IdP with username "me", then I think people just won't be able to interact with their user in our product if we assume "me" is special.

But I think this comment might not be right about the current implementation of this endpoint.

Reply:
thats fair, im fine w session/me then